src/pages/ReduxToolkitTodo/redusers/rtkReducers.js

```diff
@@ -0,0 +1,80 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { v4 as uuid } from "uuid";
+
+const initialState = {
+  rtkTodos: [],
+};
+
+const todoSlice = createSlice({
+  name: "rtkTodo",
+  initialState,
+  reducers: {
+    createTodo: (state, { payload: taskData }) => {
+      const newTask = {
+        id: uuid(),
+        text: taskData.taskText,
+        description: taskData.taskDescription,
+        isCompleted: false,
+        isEditable: false,
+      };
+
+      state.rtkTodos.unshift(newTask);
+
+      return state;
```

Comment:
you don't need to return state. you mutate it directly in rtk. remove it here and in the all same places